docs/changelog_de.md

```diff
@@ -3,6 +3,7 @@
 ## v1.1.0 (TBA)
 ### General
 * Fix #144: Die Rüstung "Gomez'Rüstung" heißt nun korrekt "Gomez' Rüstung".
+* Fix [#146](https://github.com/AmProsius/gothic-1-community-patch/pull/146): Die Rüstung "Novizen Rock" heißt nun korrekt "Novizenrock".
```

Comment:
I was against links in the changelog at first, because it would become hard to read if displayed in plain text. But I now think that is very unlikely and people will mostly view this file fully formatted here on Github.

Linking to the details of each fix is a really nice idea, to lower the threshold for users to get into the issue tracker on Github and for them to get more details in general. I would propose, however, to shorten the link to https://g1cp.org/issues/146. That link works irrespective of whether the fix is listed as issue or has been turned into a pull request.

Should we go ahead and update the entire changelog?

Reply:
> Should we go ahead and update the entire changelog?

I'd like to do so. I already wrote that on my todo list.